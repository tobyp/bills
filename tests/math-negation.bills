#!./bills
A	B.	-3
C	D.	-5--7
