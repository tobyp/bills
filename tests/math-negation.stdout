todo: A pays 3.0 to B
todo: D pays 2.0 to C
